src/SignalR/clients/java/signalr/src/test/java/com/microsoft/signalr/HubConnectionTest.java

```diff
@@ -954,6 +954,42 @@ public void invokeWaitsForCompletionMessage() {
 
         assertEquals(Integer.valueOf(42), result.timeout(1000, TimeUnit.MILLISECONDS).blockingGet());
     }
+    
+    @Test
+    public void canSendNullArgInInvocation() {
+        MockTransport mockTransport = new MockTransport();
+        HubConnection hubConnection = TestUtils.createHubConnection("http://example.com", mockTransport);
+
+        hubConnection.start().timeout(1, TimeUnit.SECONDS).blockingAwait();
+
+        AtomicBoolean done = new AtomicBoolean();
+        Single<String> result = hubConnection.invoke(String.class, "fixedMessage", null);
+        result.doOnSuccess(value -> done.set(true));
+        assertEquals("{\"type\":1,\"invocationId\":\"1\",\"target\":\"fixedMessage\",\"arguments\":[null]}" + RECORD_SEPARATOR, mockTransport.getSentMessages()[1]);
+        assertFalse(done.get());
+
+        mockTransport.receiveMessage("{\"type\":3,\"invocationId\":\"1\",\"result\":\"Hello World\"}" + RECORD_SEPARATOR);
+
+        assertEquals("Hello World", result.timeout(1000, TimeUnit.MILLISECONDS).blockingGet());
+    }
+
+    @Test
+    public void canSendMultipleNullArgsInInvocation() {
+        MockTransport mockTransport = new MockTransport();
+        HubConnection hubConnection = TestUtils.createHubConnection("http://example.com", mockTransport);
+
+        hubConnection.start().timeout(1, TimeUnit.SECONDS).blockingAwait();
+
+        AtomicBoolean done = new AtomicBoolean();
+        Single<String> result = hubConnection.invoke(String.class, "fixedMessage", null, null);
```

Comment:
```java
public <T> Single<T> invoke(Class<T> returnType, String method, Object... args)```


This already worked before this change, right? I understand Java turning a single null value into a null `Object[]` instead of a single-element `Object[]` containing null, but two nulls should remove any ambiguity.

Reply:
>This already worked before this change, right?

Yes, this worked before. I wanted to check passing null values into the other code path of checkUploadStream